### PR TITLE
New package: DuckStudio.GitHubView version 1.0.0

### PR DESCRIPTION
**Pull request by DuckDuckStudio**
### 拉取请求使用此工具创建： [komac](https://github.com/russellbanks/Komac) v2.9.0 :rocket:

@fjwxzde Please review this change. :)

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/201719)

### CODE REVIEW

File: manifests/d/DuckStudio/GitHubView/1.0.0/DuckStudio.GitHubView.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Tags:
+- 命令
+- 命令行工具
+- 终端
+- 控制台
+- 终端工具
+- 命令工具
+- 信息查看
+- github
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/GitHubView/releases/tag/1.0.0
```
Comment (DuckDuckStudio) [inline manifests/d/DuckStudio/GitHubView/1.0.0/DuckStudio.GitHubView.locale.zh-CN.yaml:28]:
```suggestion
ReleaseNotesUrl: https://github.com/DuckDuckStudio/GitHubView/releases/tag/1.0.0
Documentations: 
- DocumentLabel: README
  DocumentUrl: https://github.com/DuckDuckStudio/GitHubView/blob/master/README.md
- DocumentLabel: 贡献指南
  DocumentUrl: https://github.com/DuckDuckStudio/GitHubView/blob/master/CONTRIBUTING.md
```

**Review by fjwxzde — APPROVED** [review]

**Comment by DuckDuckStudio** [comment]
Retry:
image

File: manifests/d/DuckStudio/GitHubView/1.0.0/DuckStudio.GitHubView.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- ghv
+Protocols:
+- https
+ReleaseDate: 2024-12-23
```
Comment (DuckDuckStudio) [inline manifests/d/DuckStudio/GitHubView/1.0.0/DuckStudio.GitHubView.installer.yaml:15]:
```suggestion
ReleaseDate: 2024-12-23
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.DotNet.Runtime.8
```

**Comment by stephengillie** [comment]
Verify:
image
Unfortunately, the VM isn't well-prepared to display the characters.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**